### DISCUSSION TITLE
One more button/link could turbo charge VideoDownloader !

### DISCUSSION THREAD

**Discussion by ZeroZero1think** [aclap-dev/video-downloadhelper#1475] [Help and Q&A]
### Your question

Love the product, been using it for years, the update was fantastic. Today my suspect brain told me that if VideoDownloader had a way to access the download folder where the videos are default located - WITHOUT any activity, this could mean that from this window the videos from previous sessions would  be available WITHIN the app (no need for windows fiddling). That would mean that VD could become a natural place to hop in and out of videos as one surfs, rather than just a downloader. 

### Addon & CoApp version and details

Love the product, been using it for years, the update was fantastic. Today my suspect brain told me that if VideoDownloader had a way to access the download folder where the videos are default located - WITHOUT any activity, this could mean that from this window the videos from previous sessions would  be available WITHIN the app (no need for windows fiddling). That would mean that VD could become a natural place to hop in and out of videos as one surfs, rather than just a downloader.

**Comment by jcfrog** [comment]
Hi,
Not sure to exactly understand, isn't it something like History button?

<img width="354" alt="image" src="https://github.com/user-attachments/assets/5d1218ea-974a-407b-b202-3fbda3b09c38">

**Comment by ZeroZero1think** [comment]
No. Try this. Open a webpage (I am on firefox) . Try to load a file you downloaded to your default location, yesterday. You can't. If you could open the folder, without necessarily editing, you could use this opportunity to engage with all your video library. I often download utube piano tutorial videos - so that I can use the A- B repeat function of my media player. I  can only access material downloaded in a single browser session using VD. This means I have to wander around Windows, when it would be better to have the ability , within VT, to access downloads folder - then for example, delete old videos, put them in folders, etc.

**Reply by jcfrog** [reply]
You mean a file manager and viewer in VDH?

**Reply by ZeroZero1think** [reply]
I simply mean accessing the Windows file manager. If, within VDM, one could simply open the download location set in the VDM settings, which you cannot do UNTIL you have downloaded at least one video on the same browser session, then with access to this window - basic file duties can be carried out.
Z
